### PR TITLE
Docs: Persistence - add configuration example for eager init. of plugins

### PR DESCRIPTION
**Pull request by sebastianharko**
The config for eager init is complicated, a concrete example of how it looks is useful, IMO - but let me know what you think.

PR also fixes two typos.

### CODE REVIEW

File: akka-docs/src/main/paradox/scala/persistence.md [hunk 1]
```diff
@@ -1039,10 +1039,36 @@ Java
...
 `akka.persistence.journal.auto-start-journals` and `akka.persistence.snapshot-store.auto-start-snapshot-stores`.
 
+For example, if you want eager initialization for the leveldb journal plugin and the local snapshot store plugin, your configuration should look like this:  
+
+```
+akka {
+
+  extensions = [
+    akka.persistence.Persistence
+  ]
```
Comment (patriknw) [inline akka-docs/src/main/paradox/scala/persistence.md]: I'd write that as `extensions = [akka.persistence.Persistence]` 
or `extensions = ["akka.persistence.Persistence"]`

**Review by patriknw — APPROVED** [review]
LGTM, maybe a small syntax adjustment

File: akka-docs/src/main/paradox/scala/persistence.md (hunk 1 above)
Comment (sebastianharko) [inline akka-docs/src/main/paradox/scala/persistence.md]: fixing this now :-)

**Comment by sebastianharko** [comment]
@patriknw fixed :-)

**Review by raboof — APPROVED** [review]
Great!

**Pull request merged**